dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -471,7 +471,8 @@ else if ((cmd != null)
 		_setupEditTemplatePage(reqImpl, res, config, form, user);
 
 		// *********************** BEGIN GRAZIANO issue-12-dnd-template
-		boolean isDrawed = req.getAttribute(WebKeys.TEMPLATE_IS_DRAWED)!=null?(Boolean)req.getAttribute(WebKeys.TEMPLATE_IS_DRAWED):false;
+		boolean isDrawed = req.getParameter("drawed") != null ? Boolean.valueOf(req.getParameter("drawed")) :
+				req.getAttribute(WebKeys.TEMPLATE_IS_DRAWED) != null ? (Boolean)req.getAttribute(WebKeys.TEMPLATE_IS_DRAWED) : false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)